src/server.js

```diff
@@ -274,14 +275,26 @@ const globe = [
 ];
 
 app.get(`${PREFIX_API}/:${URL_PARAM_EVENT_ID}/restaurants`, function () {
-  let lat = averageGeolocation(sf).latitude.toString();
-  let long = averageGeolocation(sf).longitude.toString();
+  getEvent,
+    async (request, response) => {
+      const { event } = request;
+      const users = event.users || {};
+      response.json({
+        status: 200,
+        data: Object.values(users),
+      });
+    };
+
+  const query = datastore.createQuery("Company");
```

Comment:
What is the kind `Company`? (I'm guessing you put test data into the datastore of type `Company`?)